layout/_third-party/comments/valine.swig

```diff
@@ -1,9 +1,3 @@
-{% set leancloud_uri = '//cdn1.lncld.net/static/js/3.11.1/av-min.js' %}
-{% if theme.vendors.leancloud %}
-  {% set leancloud_uri = theme.vendors.leancloud %}
-{% endif %}
-<script src="{{ leancloud_uri }}"></script>
-
 {% set valine_uri = '//unpkg.com/valine/dist/Valine.min.js' %}
```

Comment:
I think it can be modified to be configurable here.

Reply:
i don't know the influence of the reference of 'av-min.js'. just to do the pr according to your new update.
So, u mean it's unnecessary to delete the definition of `leancloud_uri` since `leancloud_uri` can be configured by the vendor.

Reply:
I mean `valine_uri` can set a default value, providing user-defined property configuration.

Reply:
https://github.com/theme-next/hexo-theme-next/blob/10c0333f5080ccde34170f88d4020ffe6d8e1bcd/layout/_third-party/comments/valine.swig#L7-L11

this has already available.  the default value of `valine_uri` is `//unpkg.com/valine/dist/Valine.min.js`.

Reply:
Ok, it’s my mistake, I don’t have a full view of the code.
I have no other questions.

Reply:
thanks.

Reply:
You are welcome.